README.md

```diff
@@ -27,16 +27,17 @@ k8s.dolittle.studio/ -> "k8 apiserver proxy path"
 
 ```shell
 # not needed anymore
-kubectl -n system-auth port-forward <postgres-pod> 8080:80
+kubectl -n system-authentication port-forward <postgres-pod> 8080:80
 ```
+
 Create/register the OAuth 2.0 client in hydra. Make sure the go code oath client is the exact same.
 ```shell
-kubectl -n system-auth exec $(kubectl get pod -l "component=browser" -o name -n system-auth) -- hydra --endpoint http://localhost:4445 clients create --id client-id --secret client-secret -c http://local.dolittle.studio:8080/.auth/cookies/callback
+kubectl -n system-authentication exec $(kubectl get pod -l "component=Browser" -o name -n system-authentication) -- hydra --endpoint http://localhost:4445 clients create --id client-id --secret client-secret -c http://local.dolittle.studio:8080/.auth/cookies/callback
 ```
 
 List out your clients:
 ```shell
-kubectl -n system-auth exec $(kubectl get pod -l "component=hydra" -o name -n system-auth) -- hydra --endpoint http://localhost:4445 clients list
+kubectl -n system-authentication exec $(kubectl get pod -l "component=Browser" -o name -n system-authentication) -- hydra --endpoint http://localhost:4445 clients list
```

Comment:
@jakhog  Not 100% what is hydra, but Browser seems to reference it in the deployment.